index.html

```diff
@@ -16,6 +16,13 @@
     <link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon">
     <link rel="stylesheet" href="src/style.css" />
     <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap" rel="stylesheet" />
+    <!-- font style -->
+    <link href="https://fonts.googleapis.com/css?family=Allerta Stencil" rel="stylesheet">
+    <style>
```

Comment:
hey @ujjawaltyagii currently using Tailwind CSS. I think it would be great if you add this font style using tailwind css